tests/selenium_tests/test_new_excerpt.py

```diff
@@ -26,10 +26,13 @@ def test_new_excerpt(base_url, login, file_name, file_format, selenium, reload_u
     create.send_keys(Keys.RETURN)
 
     # wait until download link appears
-    selenium.find_element_by_link_text('↻ Reload')
-    element = reload_until_condition(selenium.find_element_by_class_name, "form-control")
+    selenium.find_element_by_xpath("/html/body/div/div/div[2]/div[2]/div[1]/div/div[1]/h3")
+    first_a = "/html/body/div/div/div[2]/div[2]/div[1]/div/div[2]/div[1]/div[1]/div[2]/div/div[1]/p/a"
```

Comment:
oh, this doesn't work at all, since we go to a page where we have link texts, but the first one isn't the one we're waiting on, it is the first one that is already finished, which is not what we want.

Extracting to a constant is a good idea.